CompetitionTemplate/src/test/java/competition/subsystems/elevator/commands/StopCommandTest.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.elevator.commands;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import competition.subsystems.elevator.ElevatorSubsystem;
+import edu.wpi.first.wpilibj.MockXboxControllerAdapter;
+import xbot.common.controls.actuators.mock_adapters.MockCANTalon;
+import xbot.common.injection.BaseWPITest;
+
+public class StopCommandTest extends BaseWPITest{
+	
+	StopCommand command;
+	ElevatorSubsystem elevator;
+	
+	@Override
+	public void setUp() {
+		// TODO Auto-generated method stub
+		super.setUp();
+		
+		command = injector.getInstance(StopCommand.class);
+		elevator = injector.getInstance(ElevatorSubsystem.class);
+		elevator.temporaryHack();
+	}
+	
+	@Test
+	public void testSimple() {
+		command.execute();
+	}
+	
+	@Test
+	public void verifyMovingUp() {
+		
```

Comment:
You need to have the elevator moving before this, otherwise you're not actually testing if this stops it.